sources/ecommerce-app/package.json

```diff
@@ -4,11 +4,14 @@
   "private": true,
   "proxy": "http://localhost:8080",
   "dependencies": {
-    "@craftercms/classes": "^1.2.4",
-    "@craftercms/content": "^1.2.4",
-    "@craftercms/ice": "^1.2.4",
-    "@craftercms/search": "^1.2.4",
-    "@material-ui/core": "^4.1.1",
+    "@craftercms/classes": "^2.0.1",
+    "@craftercms/content": "^2.0.1",
+    "@craftercms/experience-builder": "^1.0.0-beta.1",
+    "@craftercms/ice": "^2.0.1",
```

Comment:
What for?

Reply:
reportNavigation was being used which was not needed, removed